app/models/saved_claim.rb

```diff
@@ -148,6 +149,20 @@ def va_notification?(email_template_id)
 
   private
 
+  def validate_schema(schema)
+    JSON::Validator.fully_validate_schema(schema, { errors_as_objects: true })
+  rescue => e
+    Rails.logger.error('Error during schema validation!', { error: e.message, backtrace: e.backtrace })
+    [] # Return empty array since we don't have an error array
```

Comment:
This returns an empty array since it appears that we expect to continue from this method unless errors were returned.

Reply:
✅  always unsettling to see exceptions swallowed entirely like this.... Still, if we need to do so, this is the safe way to do it. (no `rescue Exception` here!)

appreciate the clarifying comment.

Reply:
Yeah...I originally was going to rescue it, log it, and raise it here. But all my tests failed since that's not currently the expected behavior. So this is what we have for now. We use this in a lot of classes that inherit from here so I am very hesitant to change how it interacts outside of this class.